07.bitstream.semantics.md

```diff
@@ -1921,6 +1921,7 @@ for each tile down the image.
 **maxTileHeightSb** specifies the maximum height (in units of superblocks) that can be used for a tile (to avoid making tiles with too much area).
 
 **context_update_tile_id** specifies which tile to use for the CDF update.
+It is a requirement of bitstream conformance that context_update_tile_id is less than TileCols * TileRows.
```

Comment:
The spec also defines the variable NumTiles with the same value as TileCols * TileRow. I guess you prefer to use TileCols * TileRows directly?